set bounding box to None to avoid layer cropping (#1908)
spacetelescope/jdaviz: docs/imviz/displayimages.rst

of the data there. This information is displayed in the top bar of the UI, on the
middle-right side.

Notes on GWCS
-------------

> If you *reference data* has GWCS with bounding box, any coordinates transformation

Author: ```suggestion
If your *reference data* has GWCS with a bounding box, any coordinates transformation
```

of the data there. This information is displayed in the top bar of the UI, on the
middle-right side.

> If the mouse is outside the original bounding-box of the reference data, the transformation

Reviewer: ```suggestion
For GWCS, if the mouse is outside the original bounding box of the reference data, the transformation
```

from pixels to sky coordinates is less reliable.  This is indicated by "(est.)" and the sky

> coordinates becoming gray.

Reviewer: As stated by Nadia in https://github.com/spacetelescope/gwcs/issues/240#issuecomment-516829043 , "FITS WCS always returns values even outside the validity of the WCS model."

```suggestion
coordinates becoming gray. Note that FITS WCS has no similar concept of bounding box,
thus its mouseover will not distinguish between inside or outside in display.
```

of the data there. This information is displayed in the top bar of the UI, on the
middle-right side.

Notes on GWCS
-------------

If your *reference data* has GWCS with a bounding box, any coordinates transformation
outside that bounding box is less reliable. This still applies even when you are
looking at some other data that is not the reference data if they are linked by WCS
because all transformations in glue go through the reference data. Such a situation

> is indicated by "(est.)" and the affected coordinates becoming gray.

Reviewer: ```suggestion
is indicated by "(est.)" and the affected coordinates becoming gray.

If your data of interest also has a GWCS with a bounding box, only
the mouseover data where it overlaps with the reference data's
bounding box is completely reliable.
```
